<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetSubItem</class>
 <widget class="QWidget" name="BudgetSubItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="backgroundwidget" native="true">
     <property name="styleSheet">
      <string notr="true">#backgroundwidget
{
    background-color: rgb(255, 255, 255);
    border-bottom:1px ;
    border-color: rgba(140,140,140,120);
    border-style: solid;
}

</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,1">
        <item>
         <widget class="QLabel" name="MainTitle">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>04月总预算</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>568</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="EditButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 102, 102);
font: 19pt &quot;微软雅黑&quot;;
border:none;</string>
          </property>
          <property name="text">
           <string>编辑</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="CircularDiagram" name="circulardiagram" native="true">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>260</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>90</number>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>50</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labeltext_RemainBudget">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
font: 18pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>剩余预算:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>568</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelnum_RemainBudget">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
font: 18pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>140.00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="labeltext_CurrentBudget">
               <property name="styleSheet">
                <string notr="true">color: rgb(102, 102, 102);
font: 15pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>本月预算:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>568</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelnum_CurrentBudget">
               <property name="styleSheet">
                <string notr="true">color: rgb(102, 102, 102);
font: 15pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>150.00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labeltext_CurrentExpand">
               <property name="styleSheet">
                <string notr="true">color: rgb(102, 102, 102);
font: 15pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>本月支出:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>558</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelnum_CurrentExpand">
               <property name="styleSheet">
                <string notr="true">color: rgb(102, 102, 102);
font: 15pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>10.00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircularDiagram</class>
   <extends>QWidget</extends>
   <header location="global">BaseClass/circulardiagram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
